tests/fixtures/rubrics_api_response.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
@zqian I might need to fake up the Rubric JSON response as per your request here https://github.com/tl-its-umich-edu/IPE-competencies-updater/pull/17/files#r749695686. Please expect a change here